qgepqwat2ili/__init__.py

```diff
@@ -61,6 +61,12 @@ def main(args):
         action="store_true",
         help="saves the log files next to the input/output file",
     )
+    # 11.1.2023 / 12.1.2023 add new argument --export_model_name
```

Comment:
you can remove all such comments, this kind of information is available through the git history anyway (and they are not very helpful in the long term)